src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -114,188 +212,85 @@ interface IOptionSettlementEngine {
-        uint96 exerciseAmount,
-        uint96 underlyingAmount
-    );
-
-    /**
-     * @notice Emitted when an option id is exercised and assigned to a particular claim NFT.
-     * @param claimId The claim NFT id being assigned.
-     * @param optionId The id of the option being exercised.
-     * @param amountAssigned The total amount of options contracts assigned.
-     */
-    event ExerciseAssigned(uint256 indexed claimId, uint256 indexed optionId, uint112 amountAssigned);
+    /*//////////////////////////////////////////////////////////////
+    // Data structures
+    //////////////////////////////////////////////////////////////*/
 
     /// @dev This enumeration is used to determine the type of an ERC1155 subtoken in the engine.
     enum Type {
         None,
         Option,
-        Claim
+        OptionLotClaim
     }
 
     /// @dev This struct contains the data about an options type associated with an ERC-1155 token.
     struct Option {
-        // The underlying asset to be received
+        /// @param underlyingAsset The underlying asset to be received
         address underlyingAsset;
-        // The timestamp after which this option may be exercised
-        uint40 exerciseTimestamp;
-        // The timestamp before which this option must be exercised
-        uint40 expiryTimestamp;
-        // The address of the asset needed for exercise
-        address exerciseAsset;
-        // The amount of the underlying asset contained within an option contract of this type
+        /// @param underlyingAmount The amount of the underlying asset contained within an option contract of this type
         uint96 underlyingAmount;
-        // Random seed created at the time of option type creation
-        uint160 settlementSeed;
-        // The amount of the exercise asset required to exercise this option
+        /// @param exerciseAsset The address of the asset needed for exercise
+        address exerciseAsset;
+        /// @param exerciseAmount The amount of the exercise asset required to exercise this option
         uint96 exerciseAmount;
-        // Which option was written
-        uint96 nextClaimId;
+        /// @param earliestExerciseTimestamp The timestamp after which this option may be exercised
+        uint40 earliestExerciseTimestamp;
+        /// @param expiryTimestamp The timestamp before which this option must be exercised
+        uint40 expiryTimestamp;
+        /// @param settlementSeed Random seed created at the time of option type creation
+        uint160 settlementSeed;
+        /// @param nextClaimNum Which option was written
+        uint96 nextClaimNum;
     }
 
-    /// @dev This struct contains the data about a claim ERC-1155 NFT associated with an option type.
-    struct Claim {
-        // These are 1:1 contracts with the underlying Option struct
-        // The number of contracts written in this claim
+    // TODO Review and clarify NatSpec on these next 3 claim-related structs
```

Comment:
what's meant by this TODO>?

Reply:
Just thinking you could do a pass and make sure it's accurate.

Reply:
ok will remove on next PR